sdk/servicebus/test-resources.json

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
dumb question: can you tell me what this file does?

Reply:
good question - It's an ARM template that's being used to create resources for samples, rather than setting the `EnvVars` in `tests.yml`. This is to align with other services, which are also moving towards using ARM templates, since they are easier to maintain than a growing list of `EnvVars` in the `tests.yml` file. Specifically, formrecognizer. Context [here](https://github.com/Azure/azure-sdk-for-python/pull/15694/files?file-filters%5B%5D=.yml#r538868882) and issue [here](https://github.com/Azure/azure-sdk-for-python/issues/15736).